src/orchestrator/base/IModuleManager.sol

```diff
@@ -37,24 +37,6 @@ interface IModuleManager {
     /// @param module The module's address.
     event ModuleRemoved(address indexed module);
 
-    /// @notice Event emitted when account `account` is granted role `role` for
-    ///         module `module`.
-    /// @param module The module's address.
-    /// @param role The access control role.
-    /// @param account The account the role was granted to.
-    event ModuleRoleGranted(
```

Comment:
Just out of interest
Did these Module Roles fulfill any purpose in the first place?